doc/stdlib/math.rst

```diff
@@ -164,11 +172,29 @@ Mathematical Operators
 
    The floored quotient and modulus after division. Equivalent to ``(fld(x,y), mod(x,y))``\ .
 
+.. function:: fld1(x,m)
```

Comment:
Not sure if it actually makes a difference but I think we usually put space after commas in argument list.